swarm.cabal

```diff
@@ -16,12 +16,14 @@ description:
   For developers getting oriented, Swarm's modules are organized into
   sublibraries, from inner to outer:
 
-  * utilities
-  * swarm language
-  * swarm scenario
-  * swarm game engine
-  * swarm TUI
-  * swarm app
+  * swarm-util: miscellaneous utilities
+  * swarm-lang: parsing, typechecking, etc. for the Swarm language
+  * swarm-scenario: scenario descriptions, parsing, & processing
+  * swarm-engine: game simulation
+  * swarm-doc: generating documentation
+  * swarm-tui: textual user interface
+  * swarm-web: web interface
```

Comment:
Would it be appropriate to explain here why the web interface depends on `swarm-tui`?

Reply:
Why *does* the web interface depend on `swarm-tui`?  I'm not sure I know the answer to that off the top of my head.

In any case I'm not sure this is the place to explain that, it seems like too much detail for a comment in the `.cabal` description.  I've started working on #1689 which seems like a better place for that (and we can add a link to the module guide to the `.cabal` description).

Reply:
Sure thing, sorry, I didn't realize it was waiting for a review.